4517_Sprint/04 - xander/OCP-CH4/src/com/company/Demo.java

```diff
@@ -0,0 +1,116 @@
+      populateIndiviualCollection(aeron, 42);
+      populateIndiviualCollection(sheik, 10);
+      populateIndiviualCollection(xander, 10);
+
+      List<Vinyl> KennySortedCollection;
+      KennySortedCollection = kenny.getVinylCollection().stream()
+                                .sorted(sortOnAlbumName)
+                                .collect(Collectors.toList());
+
+      System.out.println(KennySortedCollection);
+
+      List<Vinyl> AeronKpopCollection;
+      AeronKpopCollection = aeron.getVinylCollection().stream()
+                                  .filter(v -> v.getGenre().contains("kpop"))
+                                  .peek(v -> System.out.println(v.getGenre()))
+                                  .sorted(sortOnArtistName)
+                                  .collect(Collectors.toList());
+
+      System.out.println(AeronKpopCollection);
+
+      int hoaranAmountOfpopRecords = (int)haoran.getVinylCollection().stream()
+                                          .filter(v -> v.getGenre()
+                                          .contains("pop"))
+                                          .count();
+
+      System.out.println("Haoron has " + hoaranAmountOfpopRecords + " Pop records");
+
+      Optional<Vinyl> sheikRandomJazzRapRecord;
+      sheikRandomJazzRapRecord = sheik.getVinylCollection()
+                                .stream()
+                                .filter(v -> v.getGenre()
+                                .contains("jazz"))
+                                .filter(v -> v.getGenre()
+                                .contains("rap"))
+                                .findFirst();
+
+      if(sheikRandomJazzRapRecord.isPresent()) {
+        System.out.println("Xander steals " + sheikRandomJazzRapRecord.get() + " and adds it to his own collection");
+        xander.addVinylToCollection(sheikRandomJazzRapRecord.get());
+      } else {
+        System.out.println("Sheik does not own a jazz rap record, Xander fails to steal it");
+      }
+
+    }
+
+    public static void populateIndiviualCollection(Individual individual, int amount) {
+      individual.addVinylToCollection(generateVinylArray(amount));
+    }
+
+    public static Vinyl[] generateVinylArray(int amountOfVinyls) {
+      Vinyl[] vinylArray = new Vinyl[amountOfVinyls];
+      Random r = new Random();
+
+      for(int i = 0; i < amountOfVinyls; i++) {
+        vinylArray[i] = new Vinyl(getRandomAlbum(),getRandomArtist(),i % 2 == 0 ? 12 : 7,
+                  r.nextInt(3600)+1200, r.nextInt(30)+7,
+                  generateListOfRandomGenres());
+      }
+
+      return vinylArray;
+    }
+
+    public static String getRandomArtist() {
+      String[] artists = {"Michael Jackson","AC/DC","Pink Floyd","Whitney Houston","Meat Loaf","Eagles","Fleetwood Mac"
+          ,"Shania Twain","Led zeppelin", "Celine Dion", "The Beatles", "Adele", "Metallica", "Nirvana"
+          ,"Guns N' Roses", "ABBA", "Linkin Park", "Bob Marley", "Queen", "Tommpabeats"};
+
```

Comment:
Add extra tabs to have the array items below each other, for better readability.